get_phi_fv3/get_data.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# this scripts downloads test data for the physics standalone
+
+# echo on
+set -x
+
+# get name of standalone package
+cwd=`pwd`
+dirname=`basename ${cwd}`
+
+# remove preexisting data directory
+test -d ./data
+/bin/rm -rf data
+
+# get data
+wget --quiet "ftp://ftp.cscs.ch/in/put/abc/cosmo/fuo/physics_standalone/${dirname}/data.tar.gz"
```

Comment:
The ftp link should be `phifv3` instead of `get_phi_fv3`, so either change the folder name or just change ${dirname} in here

Reply:
Good catch!  I'll keep this script consistent with the other `get_data.sh` scripts by changing the folder name to `phifv3`.